pkg/azure/actuator.go

```diff
@@ -77,6 +86,8 @@ func (a *Actuator) IsValidMode() error {
 	switch mode {
 	case constants.PassthroughAnnotation:
 		return nil
+	case constants.MintAnnotation:
+		return errors.New("mint mode is invalid")
 	default:
```

Comment:
Note to reviewers: I'm assuming in Manual mode there's either no secret or no need to annotate it?

Reply:
otherwise please let me know what the manual annotation is

Reply:
Usually we handle manual mode as the default by specifying Passthrough and Mint mode in prior case statements, as you have done here. I think we just need to ensure the default case does not return an error when stsDetected. This will enable the `Exists` and `Delete` functions to function normally for stsDetected clusters.